includes/pages/admin_shifts.php

```diff
@@ -29,7 +29,7 @@ function admin_shifts()
     $session = session();
     $start = Carbon::createTimestampFromDatetime(date('Y-m-d') . 'T00:00');
     $end = $start;
-    $mode = '';
+    $mode = 'single';
```

Comment:
I would prefer `multi` here as normally before and on an event multiple shifts get created.

Reply:
done